specification/Styling/README.md

```diff
@@ -63,6 +47,25 @@ Styles are defined with JSON and expressions written in a small subset of JavaSc
 
 ## Examples
 
+_This section is non-normative_
```

Comment:
This note is too broad since some of the content in this section is normative including conditions and meta. Maybe the note should be `Examples in this section are non-normative`